projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts

```diff
@@ -117,6 +117,26 @@ describe('IgxGrid - Advanced Filtering #grid - ', () => {
             expect(GridFunctions.getAdvancedFilteringComponent(fix)).toBeNull();
         }));
 
+        it('Should set initial Advanced Filtering', fakeAsync(() => {
```

Comment:
This test is not correct. It passes even without the fix. Probably the sample has no additional actions in the toolbar.